scripts/make_pdfs.sh

```diff
@@ -0,0 +1,22 @@
+# Generate article PDFs with maybe wrong page numbers
+for folder in $( ls -d -1 ../interviews/* ../essays/* ../papers/* ); do
+ cd $folder
+ myst build --pdf
+ cd ../../scripts
+done
+# Update page numbers in article myst.ymls
+bash ./update_pages.sh 5 ../interviews/* ../essays/* ../papers/*
```

Comment:
Ok - I updated the scripts to include interviews and essays now.

They are still treated like normal papers - we will probably need different PDF templates as well as tweaks to differentiate them in the PDF table of contents. But they are at least included for now!

@taylorgibson - feel free to merge whenever you'd like.